bootstrap/bootstrap-kubestellar.sh

```diff
@@ -194,67 +248,131 @@ if [ "$folder" == "" ]; then
 fi
 
 # Ensure kcp is installed
-if [ "$(kcp_installed)" == "false" ]; then
-    if [ "$verbose" != "" ]; then
-        echo "Installing kcp..."
+echo "************************************************"
+echo "Ensure kcp is installed"
+echo "************************************************"
+if [ "$(kcp_installed)" == "true" ]; then
```

Comment:
I see that does not test whether the `kubectl-ws` plugin is on `$PATH`.  Not sure whether you want to bother.

Reply:
I will add it